### PR TITLE
Rebaase

### PR DESCRIPTION
**Pull request by eteriSokhoyan**
GrapchClust conda version update, parallelization added and new option for locarana added.

### CODE REVIEW

**Comment by eteriSokhoyan** [comment]
@bgruening  can you please check what is the problem, i don't understand why is it failing((

**Comment by bgruening** [comment]
This is wired, it even don't look like there is an error. The one test passed.

File: tools/GraphClust/GSPAN/fasta2shrep_gspan.xml [hunk 1]
```diff
@@ -23,15 +23,28 @@
 		<param name="wins" type="text" value="40,150" size="20" label=" A list of window sizes to use" help="comma separated integers"/>
 		<param name="shift" type="integer" value="30" size="7" label="The shift of the window, relative to the window size given inpercent." help="by default 30"/>
 		<param name="u" truevalue="-u" falsevalue="" checked="True"  type="boolean" label="Ignore unstable structures (RNAshapes)" help="-u"/>
-		<param name="seq_graph_t" truevalue="--seq-graph-t" falsevalue="" checked="True"  type="boolean" label="Add for each 't #' a graph which contains no structure" help="--seq-graph-t"/>
+		<param name="seq_graph_t" truevalue="-seq-graph-t" falsevalue="" checked="True"  type="boolean" label="Add for each 't #' a graph which contains no structure" help="--seq-graph-t"/>
```
Comment (bgruening) [inline tools/GraphClust/GSPAN/fasta2shrep_gspan.xml]: is it with two or one dash? If with one the help also needs to be changed,

File: tools/GraphClust/NSPDK/NSPDK_candidateClusters.xml [hunk 2]
```diff
@@ -12,7 +12,12 @@
 
         mkdir -p SVECTOR &&
 
-        cp $data_svector SVECTOR/data.svector &&
+        #for $ins in $data_svector:
+
```
Comment (bgruening) [inline tools/GraphClust/NSPDK/NSPDK_candidateClusters.xml]: The indentation is really wired here.

File: tools/GraphClust/GSPAN/fasta2shrep_gspan.xml (hunk 1 above)
Comment (eteriSokhoyan) [inline tools/GraphClust/GSPAN/fasta2shrep_gspan.xml]: actually it's with one, but i noticed that only today, so before it was working also with 2.

File: tools/GraphClust/NSPDK/NSPDK_candidateClusters.xml [hunk 3]
```diff
@@ -12,7 +12,12 @@
...
+							ln -f -s  $ins SVECTOR
```
Comment (bgruening) [inline tools/GraphClust/NSPDK/NSPDK_candidateClusters.xml]: It's a for loop and you are overwriting the SVECTOR everytime you run this look. This is a bug.

Comment (eteriSokhoyan) [inline tools/GraphClust/NSPDK/NSPDK_candidateClusters.xml]: but i don't overwrite, i'm linking the files in SVECTOR directory. no? i mean it is working correct on my local instance

Comment (bgruening) [inline tools/GraphClust/NSPDK/NSPDK_candidateClusters.xml]: I made it a little bit more obvious.

**Comment by mmiladi** [comment]
Isn't this the error?

2017-03-09 16:14:45,379 ERROR [galaxy.visualization.genomes] Error reading twobit.loc: [Errno 2] No such file or directory: '/tmp/tmpMSmN_T/galaxy-dev/tool-data/twobit.loc'
Traceback (most recent call last):
  File "/tmp/tmpMSmN_T/galaxy-dev/lib/galaxy/visualization/genomes.py", line 210, in reload_genomes
    for line in open( os.path.join( self.app.config.tool_data_path, "twobit.loc" ) ):

**Comment by mmiladi** [comment]
@eteriSokhoyan Could you please update the gspan test file? This file is 3 month old!

https://github.com/eteriSokhoyan/galaxytools/blob/rebaase/tools/GraphClust/GSPAN/test-data/GSPAN_Outputs/1.group.gspan.bz2

**Comment by bgruening** [comment]
results from liniting:
```
Linting tool /home/bag/projects/code/galaxytools/tools/GraphClust/PrepareForMlocarna/preMlocarna.xml
Applying linter tests... CHECK
.. CHECK: 1 test(s) found.
Applying linter output... CHECK
.. INFO: 5 outputs found.
Applying linter inputs... CHECK
.. INFO: Found 6 input parameters.
Applying linter help... CHECK
.. CHECK: Tool contains help section.
.. CHECK: Help contains valid reStructuredText.
Applying linter general... CHECK
.. CHECK: Tool defines a version [0.1.0].
.. CHECK: Tool defines a name [pgma_graphclust].
.. CHECK: Tool defines an id [preMloc].
.. CHECK: Tool targets 16.01 Galaxy profile.
Applying linter command... CHECK
.. INFO: Tool contains a command.
Applying linter tool_xsd... CHECK
.. INFO: File validates against XML schema.
Linting tool /home/bag/projects/code/galaxytools/tools/GraphClust/NSPDK/NSPDK_sparseVect.xml
Applying linter tests... CHECK
.. CHECK: 1 test(s) found.
Applying linter output... CHECK
.. INFO: 1 outputs found.
Applying linter inputs... WARNING
.. WARNING: Param input [gspan_file] with no format specified - 'data' format will be assumed.
.. INFO: Found 4 input parameters.
Applying linter help... CHECK
.. CHECK: Tool contains help section.
.. CHECK: Help contains valid reStructuredText.
Applying linter general... CHECK
.. CHECK: Tool defines a version [9.2].
.. CHECK: Tool defines a name [NSPDK_sparseVect].
.. CHECK: Tool defines an id [nspdk_sparse].
.. CHECK: Tool targets 16.01 Galaxy profile.
Applying linter command... CHECK
.. INFO: Tool contains a command.
Applying linter citations... CHECK
.. CHECK: Found 2 likely valid citations.
Applying linter tool_xsd... CHECK
.. INFO: File validates against XML schema.
Failed linting
Linting tool /home/bag/projects/code/galaxytools/tools/GraphClust/NSPDK/NSPDK_candidateClusters.xml
Applying linter tests... CHECK
.. CHECK: 1 test(s) found.
Applying linter output... CHECK
.. INFO: 6 outputs found.
Applying linter inputs... WARNING
.. WARNING: Param input [data_svector] with no format specified - 'data' format will be assumed.
.. INFO: Found 21 input parameters.
Applying linter help... CHECK
.. CHECK: Tool contains help section.
.. CHECK: Help contains valid reStructuredText.
Applying linter general... CHECK
.. CHECK: Tool defines a version [9.2].
.. CHECK: Tool defines a name [NSPDK_candidateClusters].
.. CHECK: Tool defines an id [NSPDK_candidateClust].
.. CHECK: Tool targets 16.01 Galaxy profile.
Applying linter command... CHECK
.. INFO: Tool contains a command.
Applying linter citations... CHECK
.. CHECK: Found 2 likely valid citations.
Applying linter tool_xsd... CHECK
.. INFO: File validates against XML schema.
Failed linting
Linting tool /home/bag/projects/code/galaxytools/tools/GraphClust/LocARNAGraphClust/locarna_best_subtree.xml
Applying linter tests... CHECK
.. CHECK: 1 test(s) found.
Applying linter output... CHECK
.. INFO: 1 outputs found.
Applying linter inputs... CHECK
.. INFO: Found 18 input parameters.
Applying linter help... CHECK
.. CHECK: Tool contains help section.
.. CHECK: Help contains valid reStructuredText.
Applying linter general... CHECK
.. CHECK: Tool defines a version [0.1.0].
.. CHECK: Tool defines a name [locarna_graphclust].
.. CHECK: Tool defines an id [locarna_best_subtree].
.. CHECK: Tool targets 16.01 Galaxy profile.
Applying linter command... CHECK
.. INFO: Tool contains a command.
Applying linter citations... CHECK
.. CHECK: Found 4 likely valid citations.
Applying linter tool_xsd... CHECK
.. INFO: File validates against XML schema.
Linting tool /home/bag/projects/code/galaxytools/tools/GraphClust/CMFinder/cmFinder.xml
Applying linter tests... CHECK
.. CHECK: 1 test(s) found.
Applying linter output... CHECK
.. INFO: 1 outputs found.
Applying linter inputs... CHECK
.. INFO: Found 4 input parameters.
Applying linter help... CHECK
.. CHECK: Tool contains help section.
.. CHECK: Help contains valid reStructuredText.
Applying linter general... CHECK
.. CHECK: Tool defines a version [0.1.0].
.. CHECK: Tool defines a name [cmfinder].
.. CHECK: Tool defines an id [cmFinder].
.. CHECK: Tool targets 16.01 Galaxy profile.
Applying linter command... CHECK
.. INFO: Tool contains a command.
Applying linter citations... CHECK
.. CHECK: Found 3 likely valid citations.
Applying linter tool_xsd... CHECK
.. INFO: File validates against XML schema.
Linting tool /home/bag/projects/code/galaxytools/tools/GraphClust/Preprocessing/preprocessing.xml
Applying linter tests... CHECK
.. CHECK: 1 test(s) found.
Applying linter output... CHECK
.. INFO: 5 outputs found.
Applying linter inputs... CHECK
.. INFO: Found 4 input parameters.
Applying linter help... CHECK
.. CHECK: Tool contains help section.
.. CHECK: Help contains valid reStructuredText.
Applying linter general... CHECK
.. CHECK: Tool defines a version [0.1].
.. CHECK: Tool defines a name [Preprocessing].
.. CHECK: Tool defines an id [preproc].
.. CHECK: Tool targets 16.01 Galaxy profile.
Applying linter command... CHECK
.. INFO: Tool contains a command.
Applying linter citations... CHECK
.. CHECK: Found 1 likely valid citations.
Applying linter tool_xsd... CHECK
.. INFO: File validates against XML schema.
Linting tool /home/bag/projects/code/galaxytools/tools/GraphClust/GSPAN/fasta2shrep_gspan.xml
Applying linter tests... CHECK
.. CHECK: 1 test(s) found.
Applying linter output... CHECK
.. INFO: 1 outputs found.
Applying linter inputs... CHECK
.. INFO: Found 11 input parameters.
Applying linter help... CHECK
.. CHECK: Tool contains help section.
.. CHECK: Help contains valid reStructuredText.
Applying linter general... CHECK
.. CHECK: Tool defines a version [0.1].
.. CHECK: Tool defines a name [fasta_to_gspan].
.. CHECK: Tool defines an id [gspan].
.. CHECK: Tool targets 16.01 Galaxy profile.
Applying linter command... CHECK
.. INFO: Tool contains a command.
Applying linter citations... CHECK
.. CHECK: Found 2 likely valid citations.
Applying linter tool_xsd... CHECK
.. INFO: File validates against XML schema.
Linting tool /home/bag/projects/code/galaxytools/tools/GraphClust/CollectResults/glob_report.xml
Applying linter tests... CHECK
.. CHECK: 1 test(s) found.
Applying linter output... CHECK
.. INFO: 11 outputs found.
Applying linter inputs... CHECK
.. INFO: Found 27 input parameters.
Applying linter help... CHECK
.. CHECK: Tool contains help section.
.. CHECK: Help contains valid reStructuredText.
Applying linter general... CHECK
.. CHECK: Tool defines a version [0.1].
.. CHECK: Tool defines a name [cluster_collection_report].
.. CHECK: Tool defines an id [glob_report].
.. CHECK: Tool targets 16.01 Galaxy profile.
Applying linter command... CHECK
.. INFO: Tool contains a command.
Applying linter citations... CHECK
.. CHECK: Found 1 likely valid citations.
Applying linter tool_xsd... CHECK
.. INFO: File validates against XML schema.
Linting tool /home/bag/projects/code/galaxytools/tools/GraphClust/Plotting/MotifFinderPlot.xml
Applying linter tests... CHECK
.. CHECK: 1 test(s) found.
Applying linter output... CHECK
.. INFO: 1 outputs found.
Applying linter inputs... CHECK
.. INFO: Found 1 input parameters.
Applying linter help... WARNING
.. WARNING: Help section appears to be empty.
Applying linter general... CHECK
.. CHECK: Tool defines a version [0.1].
.. CHECK: Tool defines a name [Motif_Finder_Plot].
.. CHECK: Tool defines an id [motifFinderPlot].
.. CHECK: Tool targets 16.01 Galaxy profile.
Applying linter command... CHECK
.. INFO: Tool contains a command.
Applying linter tool_xsd... CHECK
.. INFO: File validates against XML schema.
Failed linting
```

**Pull request merged**